docs/develop/convert-javascript-to-typescript.md

```diff
@@ -42,22 +42,29 @@ You can use the Office Add-in template in Visual Studio to create an add-in that
 
 1. Find the **Home.js** file and rename it to **Home.ts**.
 
-2. From the **Tools** tab, choose **NuGet Package Manager** and then select **Manage NuGet Packages for Solution...**.
+2. Find the **./Functions/FunctionFile.js** file and rename it to **FunctionFile.ts**.
 
-3. With the **Browse** tab selected, enter **office-js.TypeScript.DefinitelyTyped** into the search box. Install or update this package if it is already installed. This will add the TypeScript type definitions for the Office.js library to your project.
+3. Find the **./Scripts/MessageBanner.js** file and rename it to **MessageBanner.ts**.
 
-4. In the same search box, enter **jquery.TypeScript.DefinitelyTyped**. Install or update this package if it is already installed. This will add the jQuery TypeScript definitions into your project. The packages for both jQuery and Office.js will now appear in a new file generated by Visual Studio, called **packages.config**.
+4. From the **Tools** tab, choose **NuGet Package Manager** and then select **Manage NuGet Packages for Solution...**.
+
+5. With the **Browse** tab selected, enter **office-js.TypeScript.DefinitelyTyped** into the search box. Install or update this package if it is already installed. This will add the TypeScript type definitions for the Office.js library to your project.
+
+6. In the same search box, enter **jquery.TypeScript.DefinitelyTyped**. Install or update this package if it is already installed. This will add the jQuery TypeScript definitions into your project. The packages for both jQuery and Office.js will now appear in a new file generated by Visual Studio, called **packages.config**.
 
     > [!NOTE]
     > In your TypeScript project, you can have a mix of TypeScript and JavaScript files and your project will compile. This is because TypeScript is a typed superset of JavaScript that compiles JavaScript.
 
-5. Open the **Home.ts** file and add the following declaration at the top of the file:
+7. In **Home.ts**, find the line `if(!Office.context.requirements.isSetSupported('ExcelApi', '1.1') {` and replace it with the following:
 
     ```TypeScript
-    declare var fabric: any;
+    if(!Office.context.requirements.isSetSupported('ExcelApi', 1.1)) {
     ```
 
-6. In the **Home.ts** file, find the line `Office.initialize = function (reason) {` and add a line immediately after it to polyfill the global `window.Promise`, as shown here:
+    > [!NOTE]
+    > Currently, for the project to compile successfully after it's converted to TypeScript, you must specify the requirement set number as a numeric value as shown in the previous code snippet. Unfortunately this means you'll be unable to use `isSetSupported` to test for requirement set `1.10` support, as the numeric value `1.10` evaluates to `1.1` at runtime. This article will be updated when this issue is resolved.
```

Comment:
Should we specify this issue is due to a versioning problem? If so, we could also call out the out-of-date types in this section.